howto/port/install-as-system-app.md

```diff
@@ -0,0 +1,75 @@
+Normally, a user app is signed by the developer and restricted to fewer permissions at runtime while a system app is usually [signed with the platform key](https://source.android.com/devices/tech/ota/sign_builds) when building an Android image, it's pre-installed under the system partition and running with some ["signature" protection level permissions](https://developer.android.com/guide/topics/manifest/permission-element.html#plevel) in the system process of Android container. An application must be running as a system app if it
+
+- requires to access Android hidden APIs
+- gains some signature protection level permissions
+
+Anbox Cloud allows a user app to be installed as a system app in Android container. This can be done by using the `aam install-system-app` command.
```

Comment:
Shouldn't we give a quick summary here of the steps necessary we then described further below?